[2024-12-31T16:41:33.547+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T16:40:58.613929+00:00 [queued]>
[2024-12-31T16:41:33.564+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T16:40:58.613929+00:00 [queued]>
[2024-12-31T16:41:33.565+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-31T16:41:33.966+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_clear_all_data> on 2024-12-31 16:40:58.613929+00:00
[2024-12-31T16:41:33.984+0000] {standard_task_runner.py:60} INFO - Started process 994 to run task
[2024-12-31T16:41:33.988+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_clear_all_data', 'manual__2024-12-31T16:40:58.613929+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpdioiohck']
[2024-12-31T16:41:33.992+0000] {standard_task_runner.py:88} INFO - Job 153: Subtask call_clear_all_data
[2024-12-31T16:41:34.056+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T16:40:58.613929+00:00 [running]> on host 403e7aadd7ee
[2024-12-31T16:41:34.149+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_clear_all_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T16:40:58.613929+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-31T16:40:58.613929+00:00'
[2024-12-31T16:41:34.151+0000] {sql.py:276} INFO - Executing: CALL stage.clear_all_data()
[2024-12-31T16:41:34.160+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-31T16:41:34.169+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-31T16:41:34.179+0000] {sql.py:450} INFO - Running statement: CALL stage.clear_all_data(), parameters: None
[2024-12-31T16:41:34.230+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_clear_all_data, execution_date=20241231T164058, start_date=20241231T164133, end_date=20241231T164134
[2024-12-31T16:41:34.283+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-31T16:41:34.312+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
